### PR TITLE
docs: change the localhost by the react native hostname.

### PR DESCRIPTION
**Pull request by HammzaHM**

### CODE REVIEW

**Comment by Simek** [comment]
Thanks for the follow up @HammzaHM! 👍 Let's use relative links, and skip a domain.

File: website/versioned_docs/version-0.76/debugging.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ React Native DevTools replaces the previous Flipper, Experimental Debugger, and
 
 We continue to offer legacy debugging methods such as Direct JSC Debugging and Remote JS Debugging (deprecated) (see [Other Debugging Methods](./other-debugging-methods)).
 
-React Native DevTools is designed for debugging React app concerns, and not to replace native tools. If you want to inspect React Native’s underlying platform layers (for example, while developing a Native Module), please use the debugging tools available in Xcode and Android Studio (see [Debugging Native Code](http://localhost:3000/docs/next/debugging-native-code)).
+React Native DevTools is designed for debugging React app concerns, and not to replace native tools. If you want to inspect React Native’s underlying platform layers (for example, while developing a Native Module), please use the debugging tools available in Xcode and Android Studio (see [Debugging Native Code](https://reactnative.dev/docs/next/debugging-native-code)).
```
Comment (cortinico) [inline website/versioned_docs/version-0.76/debugging.md:52]:
```suggestion
React Native DevTools is designed for debugging React app concerns, and not to replace native tools. If you want to inspect React Native’s underlying platform layers (for example, while developing a Native Module), please use the debugging tools available in Xcode and Android Studio (see [Debugging Native Code](/docs/next/debugging-native-code)).
```

File: website/versioned_docs/version-0.76/react-native-devtools.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ React Native DevTools is our new debugging experience featuring an end-to-end re
 
 ![React Native DevTools opened to the "Welcome" pane](/docs/assets/debugging-rndt-welcome.jpg)
 
-React Native DevTools is designed for debugging React app concerns, and not to replace native tools. If you want to inspect React Native’s underlying platform layers (for example, while developing a Native Module), please use the debugging tools available in Android Studio and Xcode (see [Debugging Native Code](http://localhost:3000/docs/next/debugging-native-code)).
+React Native DevTools is designed for debugging React app concerns, and not to replace native tools. If you want to inspect React Native’s underlying platform layers (for example, while developing a Native Module), please use the debugging tools available in Android Studio and Xcode (see [Debugging Native Code](https://reactnative.dev/docs/next/debugging-native-code)).
```
Comment (cortinico) [inline website/versioned_docs/version-0.76/react-native-devtools.md:12]:
```suggestion
React Native DevTools is designed for debugging React app concerns, and not to replace native tools. If you want to inspect React Native’s underlying platform layers (for example, while developing a Native Module), please use the debugging tools available in Android Studio and Xcode (see [Debugging Native Code](/docs/next/debugging-native-code)).
```

**Review by cortinico — CHANGES_REQUESTED** [review]

File: website/versioned_docs/version-0.76/react-native-devtools.md (hunk 2 above)
Comment (HammzaHM) [reply website/versioned_docs/version-0.76/react-native-devtools.md]: @cortinico Thanks for your request changes.

**Review by cortinico — APPROVED** [review]

**Pull request merged**